turbine/src/broadcast_stage/broadcast_utils.rs

```diff
@@ -96,6 +99,34 @@ pub(super) fn recv_slot_entries(receiver: &Receiver<WorkingBankEntry>) -> Result
     })
 }
 
+// Returns the Merkle root of the last erasure batch of the parent slot.
+pub(super) fn get_chained_merkle_root(
+    slot: Slot,
+    parent: Slot,
+    blockstore: &Blockstore,
+) -> Result<Hash> {
+    if slot == parent {
+        debug_assert_eq!(slot, 0u64);
+        return Ok(Hash::default());
+    }
+    debug_assert!(parent < slot, "parent: {parent} >= slot: {slot}");
```

Comment:
any reason to only make these debug?

Reply:
I thought that if the this does not fail in tests then we should be good.
So the extra `panic` code in release builds is unnecessary.